### ISSUE TITLE
Q&A: Defect: Question-picker modal sometimes opens halfway off the screen

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#14605]
### Expected behavior

In the Q&A Group UI (e.g., for R&S Detail page or Benefit Details page), when I open the Q&A picker to add Q&S, the modal should be fully visible on the screen.

### Observed behavior

The modal appears halfway off the screen, so i can't see some of the questions or the buttons.

- Laura's [comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10941#issuecomment-1651719344) with screen shot
- So far only Laura can reproduce, on Chrome. (See comment thread above.)

## Acceptance criteria

>Consider
> - Design / Accessibility reviews
> - Collab cycle requirements
> - Device sizes (mobile first)
> - Documentation updates / Change management
< - [Content model documentation](https://prod.cms.va.gov/admin/structure/cm_document)
> - Testing notes

- [ ]

**Comment by jilladams** [comment]
@laflannery can you confirm you're on a Windows machine when you see this happen?

**Comment by laflannery** [comment]
@jilladams Yes I use Windows

**Comment by wesrowe** [comment]
Laura Willwerth said she has seen this a bunch of times, she's not sure if it's always the question picker modal or if it's other Drupal modals. Edge browser doesn't do it.

@laflannery asked if she should bring this to the CMS team. I think that might be a logical step since the modal is not something coded specifically for this feature. cc @jilladams @productmike

**Comment by jilladams** [comment]
Yep, should become a CMS team issue, then, sounds like. We can rewrite this one and pass it over - tweaking now.

**Comment by productmike** [comment]
@jilladams @wesrowe absorbing into out backlog now for review. May need a bit of back & forth to resolve, but we'll take ownership of that if needed. Obviously a defect we'd prefer to not let linger, but otherwise any specific things I should know for prioritization purposes?

**Comment by jilladams** [comment]
Not afaik, and just noting: this seems to have been happening for awhile and was just recently ticketed, not sure how many users are affected, etc, so I think y'all are welcome to determine priority.

**Comment by productmike** [comment]
@laflannery  to test again and let us know. Erika has reproduced

**Comment by laflannery** [comment]
Ok Here's what's curious to me, I thought it might be a width thing but my laptop with a width of 1440 this happens but on my monitor it does not so I can't understand that. Also, if I open this on my monitor, it is all good, then drag it over to my laptop it stays good!

**Comment by productmike** [comment]
@laflannery Yikes, this keeps getting deeper, lol. Obviously, this is a thing and will keep around, but for now will stay lower in priority until the status quo changes. Doesn't seem pervasive enough at this point to suck up capacity at this point.
